### PR TITLE
Weekly 404 fixes

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

404 fixes

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/search/search-query-language/search-operators/cat.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Where:
 ## Example
 
 For example: 
-
+
```
Comment (kimsauce) [inline docs/search/search-query-language/search-operators/cat.md:23]:
```suggestion
```

File: docs/search/search-query-language/search-operators/cat.md [hunk 2]
```diff
@@ -20,7 +20,6 @@ Where:
 ## Example
 
 For example:
```
Comment (kimsauce) [inline docs/search/search-query-language/search-operators/cat.md:22]:
```suggestion
For example:

```

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by kimsauce** [comment]
> LGTM

Thank you for the review!

**Pull request merged**